packages/backend/src/libp2p/connectionsManager.tor.test.ts

```diff
@@ -59,4 +79,48 @@ describe('Connections manager', () => {
     expect(await spyOnDestroyHiddenService.mock.results[0].value).toBeTruthy()
     await connectionsManager.closeAllServices()
   })
+
+  it('dials many peers on start', async () => {
+    const store = prepareStore().store
+    const factory = await getFactory(store)
+    const community = await factory.create<ReturnType<typeof communities.actions.addNewCommunity>['payload']>('Community')
+    const userIdentity = await factory.create<ReturnType<typeof identity.actions.addNewIdentity>['payload']>(
+      'Identity', { id: community.id, nickname: 'john' }
+    )
+    const ports = await getPorts()
+
+    connectionsManager = new ConnectionsManager({
+      torBinaryPath: '../../3rd-party/tor/linux/tor',
+      torResourcesPath: '../../3rd-party/tor/linux',
+      socketIOPort: ports.socksPort,
+      options: {
+        env: {
+          appDataPath: tmpAppDataPath
+        },
+      }
+    })
+    await connectionsManager.init()
+    const spyOnDial = jest.spyOn(WebSockets.prototype, 'dial')
```

Comment:
would be nice to make some of the `dial` calls return rejected promise

Reply:
Right now it's the other way around - all are rejected because I'm using wrong onion address:
```
  backend:libp2p:websockets:err connection error: Unexpected server response: 502 +1ms
  backend:libp2p:websockets:err error connecting to /dns4/ov76pufitw.onion/tcp/443/wss/p2p/QmZCJznNnWekSrvPoDg1vRGf6ywKTTD8tEWYTWdmCx4N7Z. Details: Unexpected server response: 502 +0ms
```

Reply:
ah, I thought spyOn also mocks, but I see it doesnt